pom.xml

```diff
@@ -82,7 +82,7 @@ flexible messaging model and an intuitive client API.</description>
     <pulsar.broker.compiler.release>${maven.compiler.target}</pulsar.broker.compiler.release>
     <pulsar.client.compiler.release>8</pulsar.client.compiler.release>
 
-    <pulsar.client.python.version>2.10.1</pulsar.client.python.version>
+    <pulsar.client.python.version>3.2.0</pulsar.client.python.version>
```

Comment:
I think we need to wait for https://lists.apache.org/thread/cc1yd3nqmmhr54jxnvrtwrg2kph3nf2z, right?

Reply:
Yes, that's right. I was going to do 3.1.0, but since the python client release should finish before we go for the pulsar 3.1.0 release, I decided to make this 3.2.0.